packages/flutter_driver/lib/src/extension/extension.dart

```diff
@@ -298,6 +301,21 @@ class FlutterDriverExtension {
       await _waitUntilFrame(() => SchedulerBinding.instance.transientCallbackCount == 0);
   }
 
+  Future<GetSemanticsIdResult> _getSemanticsId(Command command) async {
+    final GetSemanticsId semanticsCommand = command;
+    final Finder target = await _waitForElement(_createFinder(semanticsCommand.finder));
+    final Element element = target.evaluate().single;
+    RenderObject renderObject = element.renderObject;
+    SemanticsNode node;
+    while (renderObject != null && node == null) {
+      node = renderObject.debugSemantics;
```

Comment:
Just to be sure: It's guaranteed that drive tests always run with asserts enabled? Otherwise debugSemantics would be null even when semantics are enabled.

Reply:
Not always (microbenchmarks), but for these integration tests we'll always be using a debug build.  Added doc comment to the command.